src/docfx/localization/LocalizationUtility.cs

```diff
@@ -131,11 +131,12 @@ public static bool TryGetSourceRepository(string remote, string branch, out stri
                     sourceBranch = contributionBranch;
                 }
 
-                if (sourceBranch != "live")
+                if (sourceBranch != "master" &&
+                    !GitUtility.RemoteBranchExists(sourceRemote, sourceBranch))
```

Comment:
`ls-remote` is a network call, better not to access network in a utility function, this function is used in build as well.

Reply:
docfx make sure the source repo are only get once during restore and build. so it's should be OK